cli/src/main/java/org/schlunzis/vigilia/cli/commands/VigCommand.java

```diff
@@ -0,0 +1,86 @@
+package org.schlunzis.vigilia.cli.commands;
+
+import lombok.CustomLog;
+import org.schlunzis.vigilia.cli.ui.InputPresenter;
+import org.schlunzis.vigilia.cli.ui.OptionPresenter;
+import picocli.CommandLine;
+
+@CustomLog
+@CommandLine.Command(
+        name = "vig",
+        mixinStandardHelpOptions = true,
+        versionProvider = VersionProvider.class,
+        description = "CLI for the vigilia ecosystem",
+        subcommands = {
+                IndexCommand.class,
```

Comment:
Some complex structures possible I see. Subcommands of Subcommands of Subcommands and so on.

Reply:
I like